src/components/common/QuestionnaireLabel/NetPromoter/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import QuestionnaireLabel from '../QuestionnaireLabel';
+import NetPromoterContent from './NetPromoterContent';
+
+const NetPromoter = () => {
+  return <QuestionnaireLabel contentComponent={<NetPromoterContent />} />;
```

Comment:
Suggestion: 要不試試看 children 🥺 
```suggestion
  return (
    <QuestionnaireLabel>
      <NetPromoterContent />
    </QuestionnaireLabel>
  );
```

Reply:
好的，我再重新修改看看

Reply:
已改成傳入 children